boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -297,6 +325,18 @@ TEST_F(FsmStateMachineTest, check_brake_pressure_can_signals_in_all_states)
         CANMSGS_FSM_NON_CRITICAL_ERRORS_BRAKE_PRESSURE_OUT_OF_RANGE_OVERFLOW_CHOICE);
 }
 
+// FSM-18
+TEST_F(
+    FsmStateMachineTest,
+    check_brake_pressure_actuation_can_signals_in_all_states)
+{
+    CheckBinaryStatusCanSignalsInAllStates(
```

Comment:
Since this is only used once, I don't really see the point of a function? I would just get rid of `CheckBinaryStatusCanSignalsInAllStates` to remove a level of indirection

Reply:
I added `CheckBinaryStatusCanSignalsInAllStates` for when I add a fxn to get the BRAKE_OC_SC_OK status. Should I still get rid of it for this PR?

Reply:
When did you add that? I don't see it

Reply:
Oh i meant for later